src/Perpetuum/Services/RiftSystem/Rift.cs

```diff
@@ -199,7 +202,8 @@ public void UseItem(Player player)
             {
                 var destZone = player.Character.GetZone(8); // Hershfield zone
                 var teleport = _teleportStrategyFactories.TeleportToAnotherZoneFactory(destZone);
-                teleport.TargetPosition = new Position(964, 804); // At the Hershfield main terminal
+                var dockingBase = _dockingBaseHelper.GetDockingBase(142); // Hershfield main terminal
```

Comment:
Ok while this does get the terminal as intended - we now have 2 hardcoded values and a new dependency.
Seeing this now, let's see if we can avoid one of these and "find" the other.
The zone has the terminal as a Unit on it...
Or
The dockingbase is surely a member of a Zone...
If you can resolve the Zone from the DockingBase - that would probably be the best way forward.

Whichever strategy you pick - as long as we dont have to know the ID of both the zone AND the terminal - that would be best.

Reply:
I updated the code, but the only thing I am not sure in terms of performance is the DockingBase explicit cast.

Reply:
It turns out, the OfType method was more convenient. I did an other commit.